pkg/k8shandler/util_test.go

```diff
@@ -104,3 +106,98 @@ func TestSelectorsCommonOverwritten(t *testing.T) {
 		t.Errorf("Expected %v but got %v", expected, actual)
 	}
 }
+
+func TestNoRedundancyPolicySpecified(t *testing.T) {
+
+	esCR := &api.Elasticsearch{
+		Spec: api.ElasticsearchSpec{},
+	}
+
+	isValid := isValidRedundancyPolicy(esCR)
+
+	if !isValid {
+		t.Errorf("Users don't have to select redundancy policy. SingleRedundancy is the default")
```

Comment:
"Expected default policy of SingleRedundancy to be used, incorrectly flagged as invalid"